owncloud/owncloud.py

```diff
@@ -717,6 +717,34 @@ def delete_user(self, user_name):
 
         raise ResponseError(res)
 
+    def get_user_list(self, user_name):
+        """Gets a lit of users via provisioning API.
+        If you get back an error 999, then the provisioning API is not enabled.
+
+        :param user_name:  substring of username to be checked 
+        :returns: a list of users matching the user_name sub string
+        :raises: ResponseError in case an HTTP error status was returned
+
+        """
+
+        res = self.__make_ocs_request(
+            'GET',
+            self.OCS_SERVICE_CLOUD,
+            'users?search=' + user_name
```

Comment:
No, you should url encode it, see https://github.com/owncloud/pyocclient/blob/master/owncloud/owncloud.py#L650

URL encode means that spaces get converted to "%20", slashes to "%2F", etc.
This is not done automatically by the "requests" lib if you simply concatenate strings like you do here.